src/runtime/vdom/vdom-render.ts

```diff
@@ -695,7 +695,7 @@ export const patch = (oldVNode: d.VNode, newVNode: d.VNode) => {
  *
  * @param elm the element of interest
  */
-const updateFallbackSlotVisibility = (elm: d.RenderNode) => {
+export const updateFallbackSlotVisibility = (elm: d.RenderNode) => {
```

Comment:
Had to export this to use in the patches in `dom-extras`. Can move this somewhere else if we want